HotFix/Core/Session.cs

```diff
@@ -81,6 +93,8 @@ public bool Receive()
                     if (inbound[35].Is("1")) HandleTestRequest(configuration, state, channel, inbound, outbound);
                     if (inbound[35].Is("2")) HandleResendRequest(configuration, state, channel, inbound, outbound);
                     if (inbound[35].Is("4")) HandleSequenceReset(configuration, state, channel, inbound, outbound);
+                    if (inbound[35].Is("5")) Active = false;
```

Comment:
The code above uses public methods with 5 parameters each, regardless of whether they use them or not. Should I make them like that?

Ie:
```csharp
public void HandleLogout(IConfiguration configuration, State state, Channel channel, FIXMessage inbound, FIXMessageWriter outbound);

public void HandleLogon(IConfiguration configuration, State state, Channel channel, FIXMessage inbound, FIXMessageWriter outbound);
```